extensions/Worker.Extensions.CosmosDB/src/Worker.Extensions.CosmosDB.csproj

```diff
@@ -17,6 +17,8 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
+    <ProjectReference Include="..\..\..\src\DotNetWorker.Core\DotNetWorker.Core.csproj" />
+    <PackageReference Include="Azure.Cosmos" Version="4.0.0-preview3" />
```

Comment:
CosmosDB v4 has not been actively developed for a while. While [v3](https://www.nuget.org/packages/Microsoft.Azure.Cosmos) is still being worked on. Should we use v3 instead? Or does this extension only work with v4?

Reply:
Let me get back to you on this one. When we started this work we always had the intension of targeting v4 (was supposed to be released in November). I can sync with the v-team on this topic

Reply:
I had a chat with the Cosmos team and we should indeed be using v3!